feat: add docker-compose setting for single node (#18)
Finschia/tutorials: localnet/single/README.md

## Accounts

> Through setup.sh, 11 accounts added to genesis. Every account are derived from same mnemonic (`mind flame tobacco sense move hammer drift crime ring globe art gaze cinnamon helmet cruise special produce notable negative wait path scrap recall have`) and every account have same amount of balances (`100000000000cony`).

Reviewer: ```suggestion
Through setup.sh, 11 accounts added to genesis. Every account is derived from the same mnemonic (`mind flame tobacco sense move hammer drift crime ring globe art gaze cinnamon helmet cruise special produce notable negative wait path scrap recall have`) and every account has the same amount of balances (`100000000000cony`).
```
